app/src/views/campaigns/index.vue

```diff
@@ -0,0 +1,298 @@
+<template>
+  <div class="app-container">
+    <!-- Filter -->
+    <div class="filter-container">
+      <el-input
+        v-model="listQuery.title"
+        :placeholder="$t('table.campaigns.name')"
+        style="width: 200px;"
+        class="filter-item"
+        @keyup.enter.native="handleFilter"
+      />
+      <el-button
+        v-waves
+        style="margin-left: 10px;"
+        class="filter-item"
+        type="primary"
+        icon="el-icon-search"
+        @click="handleFilter"
+      >{{ $t('table.search') }}</el-button>
+      <el-button
+        class="filter-item"
+        style="margin-left: 10px;"
+        type="primary"
+        icon="el-icon-edit"
+        @click="handleCreate"
+      >{{ $t('table.add') }}</el-button>
+      <el-button
+        v-waves
+        :loading="downloadLoading"
+        class="filter-item"
+        type="primary"
+        icon="el-icon-download"
+        @click="handleDownload"
+      >{{ $t('table.export') }}</el-button>
+    </div>
+
+    <!-- Table -->
+    <el-table
+      :key="tableKey"
+      v-loading="listLoading"
+      :data="list"
+      border
+      fit
+      highlight-current-row
+      style="width: 100%;"
+    >
+      <el-table-column :label="$t('table.campaigns.name')" min-width="150px">
+        <template slot-scope="{ row }">
+          <span class="link-type" @click="handleUpdate(row)">{{ row.name }}</span>
+        </template>
+      </el-table-column>
+      <el-table-column :label="$t('table.campaigns.startAt')" min-width="150px">
+        <template slot-scope="{ row }">
+          <span>{{ row.startAt }}</span>
```

Comment:
We will add a moment filter to display formatted date

Reply:
* vue-moment
* moment

* import in main.js
* apply filter {{ date | formate('LL') }}